google/cloud/security/common/gcp_type/firewall_rule.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A Firewall.
+
+See: https://cloud.google.com/compute/docs/reference/latest/firewalls
+"""
+
+from google.cloud.security.common.util import parser
+
+# pylint: disable=too-many-instance-attributes
+
+
+class FirewallRule(object):
+    """Represents Firewall resource."""
+
+    def __init__(self, **kwargs):
+        """Firewall resource.
+
+        Args:
+          kwargs (dict): Object properties"""
```

Comment:
I think this probably made pylint go off and you're addressing it -- but could you indent by 2 more spaces and put the """ on its own line?